openhands/resolver/README.md

```diff
@@ -0,0 +1,182 @@
+# OpenHands Github Issue Resolver 🙌
+
+Need help resolving a GitHub issue but don't have the time to do it yourself? Let an AI agent help you out!
+
+This tool allows you to use open-source AI agents based on [OpenHands](https://github.com/all-hands-ai/openhands)
+to attempt to resolve GitHub issues automatically. While it can handle multiple issues, it's primarily designed
+to help you resolve one issue at a time with high quality.
+
+Getting started is simple - just follow the instructions below.
+
+## Using the GitHub Actions Workflow
+
+This repository includes a GitHub Actions workflow that can automatically attempt to fix individual issues labeled with 'fix-me'.
+Follow these steps to use this workflow in your own repository:
+
+1. [Create a personal access token](https://github.com/settings/tokens?type=beta) with read/write scope for "contents", "issues", "pull requests", and "workflows"
+
+2. Create an API key for the [Claude API](https://www.anthropic.com/api) (recommended) or another supported LLM service
+
+3. Copy `examples/openhands-resolver.yml` to your repository's `.github/workflows/` directory
```

Comment:
This file is missing from this PR. It can be downloaded from here: https://github.com/All-Hands-AI/openhands-resolver/blob/main/examples/openhands-resolver.yml